docs/podman-system-prune.1.md

```diff
@@ -0,0 +1,38 @@
+% PODMAN(1) Podman Man Pages
+% Brent Baude
+% December 2018
+# NAME
+podman-system-prune - Remove all unused data
+
+# SYNOPSIS
+**podman system prune**
+[**-all**|**--a**]
+[**-force**|**--f**]
+[**-help**|**--h**]
+[**-volumes**|**--v**]
+
+# DESCRIPTION
+**podman system prune** removes all dangling systems from local storage. With the `all` option,
+you can delete all unused systems.  Unused systems are dangling systems as well as any system that
+does not have any containers based on it.
+
+By default, volumes are not removed to prevent important data from being deleted if there is currently no container using the volume. Use the --volumes flag when running the command to prune volumes as well.
+
```

Comment:
Can we define what we mean by **systems** (i.e., containers, images, build cache)? Same applies to the help message. Currently, it's hard to figure out which data is removed without actually running the command.